packages/web/src/scss/components/FileUploader/README.md

```diff
@@ -49,17 +49,8 @@ the available horizontal space:
 </div>
 ```
 
-### Maximum File Size (JavaScript)
-
-The maximum size of the uploaded file that is validated by the JavaScript plugin can be adjusted. The default value is
-10 MB. To increase the limit for example to 20 MB, add the `data-max-file-size` attribute:
-
-```html
-<div class="FileUploader" data-toggle="fileUploader" data-max-file-size="20000000">
-  <!-- FileUploaderInput -->
-  <!-- FileUploaderList -->
-</div>
-```
+In this way, the error message will stop being displayed when the limit of the number
```

Comment:
I assume there is an incorrect placement of this sentence. 

Also maybe reword it, so it makes sense. Feel free to consult it in person.

Reply:
deleted